package.json

```diff
@@ -143,14 +143,14 @@
     "clean": "rm -rf node_modules yarn.lock",
     "refresh": "yarn clean && yarn",
     "test": "jest",
+    "predev": "node src/directory/generateDirectory.mjs",
     "dev": "next dev",
     "spellcheck": "cspell \"src/**/*.mdx\" --no-progress",
     "spellcheck-diff": "git diff --name-only --cached | awk \"/src.*\\.mdx/{print}\" | npx cspell --no-must-find-files --file-list stdin",
-    "build": "node tasks/generate-sitemap.mjs && next build && next export -o client/www/next-build",
+    "build": "node src/directory/generateDirectory.mjs && node tasks/generate-sitemap.mjs && next build && next export -o client/www/next-build",
```

Comment:
If anyone has any suggestions for cleaning up our scripts, please let me know! I added a `predev` above so that it builds the directory object before `next dev` is ran

Our `yarn build` script is pretty big too, but not sure if it makes sense to split everything out into a `prebuild` step. Open to feedback on this

Reply:
I think you could move the generate directory command to a "prebuild" step. Then dev could become:
```
"dev": "yarn prebuild && next dev"
```